src/managers/config.py

```diff
@@ -415,14 +427,15 @@ def zk_jaas_config(self) -> str:
         Returns:
             String of Jaas config for ZooKeeper auth
         """
-        return f"""
-Client {{
-    org.apache.zookeeper.server.auth.DigestLoginModule required
-    username="{self.state.zookeeper.username}"
-    password="{self.state.zookeeper.password}";
-}};
-
+        return inspect.cleandoc(
+            f"""
+            Client {{
+                org.apache.zookeeper.server.auth.DigestLoginModule required
+                username="{self.state.zookeeper.username}"
+                password="{self.state.zookeeper.password}";
+            }};
         """
+        )
```

Comment:
praise: TIL

Usually I would use [`textwrap.dedent`](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) but it looks like a better option